packages/flutter/lib/src/widgets/service_extensions.dart

```diff
@@ -0,0 +1,86 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Service extension that, when called, will output a string representation of
+/// this app's widget tree to console.
+///
+/// See also:
+///
+/// * [BindingBase.initServiceExtensions], where the service extension is
+///   registered.
+const String debugDumpAppExt = 'debugDumpApp';
+
+/// Service extension that, when called, will overlay a performance graph on top
+/// of this app.
+///
+/// See also:
+///
+/// * [WidgetsApp.showPerformanceOverlayOverride], which is the flag
+///   that this service extension exposes.
+/// * [BindingBase.initServiceExtensions], where the service extension is
+///   registered.
+const String showPerformanceOverlayExt = 'showPerformanceOverlay';
```

Comment:
Regarding the "Ext" suffix: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-abbreviations